helm/cluster-vsphere/values.yaml

```diff
@@ -1,15 +1,3 @@
-apiServer:
-  enableAdmissionPlugins: "NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
-  featureGates: ""
-  certSANs: []
-
-cluster:
-  kubernetesVersion: "v1.25.16"
-  enableEncryptionProvider: true
-
-controllerManager:
-  featureGates: ""
-
 global:
```

Comment:
Looks like the values are missing the default `worker` nodepool
https://github.com/giantswarm/cluster-vsphere/pull/197/files#diff-91218d0fa586a5a23b86c05ebd3ed9ed502aee6ace9953024119e735d7e1557dR618

I can't add a comment but [lines 26 to 31](https://github.com/giantswarm/cluster-vsphere/pull/197/files#diff-bb01af83b1bd47474d07984492d2206e5dc6cb461bf210f0c8bcc185d94ae2a0R26-R31) were supposed to be deleted in https://github.com/giantswarm/cluster-vsphere/pull/186 🤔